strict.go

```diff
@@ -48,6 +48,16 @@ func ContentType(ctypes ...string) http.HandlerFunc {
 	}
 }
 
+// ContentLength generates a handler that writes a 411 Length Required response if the request
+// doesn't contain the Content-Length header
```

Comment:
Can you please add a trailing ".", to be consistent with other comments in the file?